priv/repo/migrations/20200518135714_login_log.exs

```diff
@@ -0,0 +1,22 @@
+defmodule Auth.Repo.Migrations.LoginLog do
+  use Ecto.Migration
+
+  def change do
+    create table(:user_agents) do
+      add :name, :string
+    end
+
+    create table(:login_logs) do
+      add :email, :binary
+      add :ip_address, :string
```

Comment:
IP Address is PII and in this case we want to store it _encrypted_ (_so that we can decrypt it in case of needing to inspect for malicious activity_).
Please consider doing: https://github.com/dwyl/fields/issues/32